_posts/2022-01-14-quorn-a-story-about-single-cell-protein.md

```diff
@@ -0,0 +1,193 @@
+3. Metabolic heat is removed using cooling water coils
+4. Waste carbon dioxide and gases are vented at the top of the bioreactor 
+5. The culture broth, now with a higher density, drops into the downcomer
+5. A portion of the biomass is harvested and the remainder returns to the bottom of the bioreactor
+
+As a side-note, the bioreactor design shown above is known as Quorn-2 and Quorn-3, and was Marlow Foods' second and third bioreactor, respectively. The first pilot bioreactor, Quorn-1, was initially used to develop ICI's Pruteen™ - a bacterial SCP meant for livestock - and looks a bit different. Quorn-2 and Quorn-3, at 150,000L, are the biggest bioreactors of their type. 
+
+This is a great design for several reasons:
+
+1. At the physical scale Quorn is dealing with, the design is more efficient AND gentler than CSTRs, whose blades would damage the growing hyphae. Mixing by aeration only is gentle and suitable for viscous broths with lots of hyphae. 
+2. The loop design avoids **short-circuiting** of fresh carbohydrate and nitrogen from the inlet directly to the outlet, which can occur in conventional CSTRs.
+2. Adding air to the bottom of a tall bioreactor allows oxygen to dissolve quickly into the media through the immense **hydrostatic pressure,** which greatly improves oxygen transfer. 
+
+The bioreactor is operated **continuously**, typically for several weeks before resetting the bioreactor (for reasons discussed later). Continuously operated bioreactors will always be more productive than batch bioreactors because they maintain a high biomass concentration at all times.
+
+### Quorn Quirks - Downstream Processing 
+
+If everything's gone well, coming out of our bioreactor is a solution of hyphae and spent substrate that's around 1.5% biomass by weight. This is where some of the problems begin to present themselves. 
+
+##### Nucleic Acids
+
+Things that grow fast and produce proteins quickly contain a lot of RNA for translation. This is a problem because humans metabolize RNA into uric acid and orotic acid, and the overaccumulation these can lead to gout and liver damage, respectively.  
+
+<div class="callout">
+💡 First note: In humans, purines are metabolized into uric acid, and pyramidines are metabolized into orotic acid. The uric acid problem is primarily related to primates: somewhere on the evolutionary calendar, a mutation in a common ancestor knocked out the gene for uricase in humans and other primates. In non-primates, uric acid is simply metabolized into allantoin, which is soluble and excreted [11]. 
+</div>
+
+The *F. venenatum* biomass has a nucleic acid content of around 8-9% w/w dry basis which is too high for human consumption. To solve this problem, the biomass is heated to 68°C for 15 minutes, and endogenous RNases break down cellular RNA into 5' ribonucleotides, which diffuse out of the cell. But this comes at a cost - over 30% of the biomass is also lost in the process.
```

Comment:
Do you know why biomass is lost?

Reply:
It seems like "cooking" the biomass at 68C just allows some intracellular components to diffuse out of the cell. I will add a note.